openhantek/src/glscope.cpp

```diff
@@ -39,9 +40,11 @@ void GlScope::fixOpenGLversion(QSurfaceFormat::RenderableType t) {
 
     // Prefer full desktop OpenGL without fixed pipeline
     QSurfaceFormat format;
-    format.setSamples(4); // Antia-Aliasing, Multisampling
+    format.setSamples(4); // Anti-Aliasing, Multisampling
     format.setProfile(QSurfaceFormat::CoreProfile);
-    if (t==QSurfaceFormat::OpenGLES) {
+
+    if (!QGLFormat::openGLVersionFlags().testFlag(QGLFormat::OpenGL_Version_3_2)
+        || t == QSurfaceFormat::OpenGLES) {
```

Comment:
The user should be able to override any detection. You need to swap those two conditions.